mlflow/tracking/fluent.py

```diff
@@ -1934,6 +1934,20 @@ def create_logged_model(
     model_type: Optional[str] = None,
     experiment_id: Optional[str] = None,
```

Comment:
Can we also update docstrings for `log_metric()`? Are there any other methods we missed?